Apromore-Core-Components/Apromore-Portal/src/main/webapp/themes/ap/common/img/icons/create_dashboard.svg

```diff
@@ -0,0 +1 @@
+<svg width="1186" height="990" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" overflow="hidden"><defs><clipPath id="clip0"><rect x="4000" y="4201" width="1186" height="990"/></clipPath></defs><g clip-path="url(#clip0)" transform="translate(-4000 -4201)"><path d="M4373.47 4794.96 4483.05 4794.96 4483.05 4987.73 4373.47 4987.73ZM4723.24 4723.99 4832.81 4773.12 4832.81 4987.73 4723.24 4987.73ZM4549.38 4618.18 4658.96 4618.18 4658.96 4989.05 4549.38 4989.05ZM4294.11 4455.91 4294.11 4610.48 4448.57 4610.48C4448.57 4695.85 4379.41 4765.05 4294.11 4765.05 4208.81 4765.05 4139.65 4695.85 4139.65 4610.48 4139.65 4525.11 4208.81 4455.91 4294.11 4455.91ZM4343.57 4404.44C4385.02 4404.43 4424.77 4420.9 4454.08 4450.21 4483.4 4479.52 4499.87 4519.28 4499.87 4560.73L4343.59 4560.73C4343.58 4508.64 4343.57 4456.54 4343.57 4404.44ZM4902.31 4303.67C4895.66 4303.67 4890.26 4309.07 4890.26 4315.72L4890.26 4400.24 4805.76 4400.24C4799.1 4400.24 4793.71 4405.63 4793.71 4412.29L4793.71 4484.71C4793.71 4491.37 4799.1 4496.76 4805.76 4496.76L4890.26 4496.76 4890.26 4581.28 4890.17 4581.28C4890.17 4587.93 4895.57 4593.33 4902.23 4593.33L4974.64 4593.33C4981.3 4593.33 4986.69 4587.93 4986.69 4581.28L4986.69 4496.76 5071.29 4496.76C5077.94 4496.76 5083.34 4491.37 5083.34 4484.71L5083.34 4412.29C5083.34 4405.63 5077.94 4400.24 5071.29 4400.24L4986.78 4400.24 4986.78 4315.72C4986.78 4309.07 4981.39 4303.67 4974.73 4303.67ZM4938.52 4201C5075.2 4201 5186 4311.81 5186 4448.5 5185.99 4585.19 5075.2 4695.99 4938.52 4696 4801.85 4696 4691.05 4585.19 4691.05 4448.5 4691.05 4311.81 4801.85 4201 4938.52 4201ZM4534.91 4201 4621.35 4201 4621.35 4290.38 4172.43 4290.38C4126.56 4290.38 4089.37 4327.57 4089.37 4373.45L4089.37 5018.55C4089.37 5064.43 4126.56 5101.62 4172.43 5101.62L4817.48 5101.62C4863.35 5101.62 4900.54 5064.43 4900.54 5018.55L4900.54 4791.4 4914.06 4793.47C4924.05 4794.48 4934.18 4795 4944.43 4795 4954.68 4795 4964.81 4794.48 4974.79 4793.47L4989.91 4791.16 4989.91 5026C4989.91 5117.13 4916.04 5191 4824.92 5191L4164.99 5191C4073.87 5191 4000 5117.13 4000 5026L4000 4366C4000 4274.88 4073.87 4201 4164.99 4201L4534.91 4201Z" fill-rule="evenodd"/></g></svg>
```

Comment:
It looks like this icon is not being used anywhere. Not sure if we need it.

Reply:
noted!. I will remove it if doesn't require during improvement in dashboard!